src/hub/models.py

```diff
@@ -72,13 +65,18 @@ class Hub(models.Model):
     paper_count = models.IntegerField(default=0)
     discussion_count = models.IntegerField(default=0)
 
-    is_removed = models.BooleanField(
-        default=False,
-        help_text=HELP_TEXT_IS_REMOVED
+    is_removed = models.BooleanField(default=False, help_text=HELP_TEXT_IS_REMOVED)
+
+    concept = models.OneToOneField(
```

Comment:
Been thinking about this a little bit, and I wonder if we should just merge these two? Like not even have the Concept model and instead put everything from the concept model into Hub.

My rationale is it makes our code less complex to have just one model representing this idea, rather than 2. 

@le-sun would like your thoughts on this too.

Reply:
I think it is good to keep these separate, at least for the time being. Concept and hub have overlap but are different enough where it makes sense to segregate the two.

For reference, we have the concept table

![image](https://github.com/ResearchHub/researchhub-backend/assets/802819/f696d400-8be0-4115-9d41-9c756d3b1cf4)

And the unified doc concept table created through the oneToOne relationship
![image](https://github.com/ResearchHub/researchhub-backend/assets/802819/7c502d46-0f72-4059-87a1-6f2ebbed1e1e)

Reply:
The part that's slightly confusing to me is having a separate unified doc concept. 

Now we have 3 places "hubs" lives and it's not evident to me what dictates our queries

Reply:
Yeah, I get your point. Three seems like a lot. 

I'll explain:

We need the unified doc association because it has **specific** attributes for that paper - such as `relevancy_score` and `level` These don't belong on either concept or hub because they are specific to the paper.

It is defined as such on `UnifiedDocumentModel`

```
    concepts = models.ManyToManyField(
        "tag.Concept",
        related_name="documents",
        blank=True,
        through="UnifiedDocumentConcepts",
    )
```

As for the concept table. It has a bunch of specific attributes to OpenAlex. I'd like to keep them separate so that their world doesn't fully influence our world - at least not quite yet but I do think there is an opportunity in the future to merge.

The good news is that the frontend will not be aware of these concepts. The FE will only be dealing with hubs.